packages/client/src/App.tsx

```diff
@@ -85,7 +90,7 @@ const MainSection = styled.section`
   background: ${({ theme }) => theme.colors.background};
 `
 
-export const routesConfig = [
+export const routesConfig: RouteObject[] = [
```

Comment:
If this is needed, can we use `satisfies` keyword instead so no type information is lost?

Reply:
In this case don't you think we matching route object's type to `RouteObject` is more important than not losing inferred types?